src/main/java/de/retest/Properties.java

```diff
@@ -273,14 +273,6 @@ public static File getDefaultSutDir() {
 		return new File( getReTestInstallDir().getParentFile(), "system-under-test" );
 	}
 
-	public static FileOutputFormat getFileOutputFormat() {
-		final String format = System.getProperty( FILE_OUTPUT_FORMAT, FileOutputFormat.ZIP.toString() );
```

Comment:
If we remove this, I assume we can also remove the property behind `FILE_OUTPUT_FORMAT`?